docs/speech-streaming.rst

```diff
@@ -0,0 +1,23 @@
+Streaming Speech Response
+=========================
+
+.. automodule:: google.cloud.speech.streaming_response
+  :members:
+  :undoc-members:
+  :show-inheritance:
+
+Streaming Speech Result
+=======================
+
+.. automodule:: google.cloud.speech.streaming_result
+  :members:
+  :undoc-members:
+  :show-inheritance:
+
+Streaming Endpointer Type
+=========================
+
+.. automodule:: google.cloud.speech.endpointer_type
```

Comment:
This module is completely empty. Shouldn't it be deleted from the docs and from the source?